### PR TITLE
Register QuantumInfo [https://github.com/BBN-Q/QuantumInfo.jl.git]

### PR DESCRIPTION
**Pull request by marcusps**

### CODE REVIEW

**Comment by tkelman** [comment]
Will need a tagged version.

**Comment by marcusps** [comment]
Now tagged.

**Comment by tkelman** [comment]
If you enabled Travis CI, would your tests pass there?

File: QuantumInfo/versions/0.0.1/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+Cliffords
+Compat
```
Comment (tkelman) [inline QuantumInfo/versions/0.0.1/requires]: best to declare a minimum version dependency that implements the features you need

**Comment by tkelman** [comment]
It's discouraged to extend Base methods like normalize, trace, etc on Base types in packages. That means importing your package can change the behavior of all other code. If you want to use the same function name, it's better not to import and change Base, but for users of the package version to qualify with `QuantumInfo.trace` etc.

**Comment by tkelman** [comment]
https://github.com/BBN-Q/QuantumInfo.jl/commit/1be50ce3b19132a41baa8a841a749435b38dae1e looks great. We can merge this now and incorporate that in the next tag if that's easier?

**Comment by marcusps** [comment]
Sure, that sounds good.

**Comment by tkelman** [comment]
Oh, and do enable Travis CI when possible. https://travis-ci.org/BBN-Q/QuantumInfo.jl

**Pull request merged**